src/modules/powerrename/dll/PowerRenameExt.vcxproj

```diff
@@ -68,7 +68,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="Resources.resx" />
+    <None Include="Resources.resx">
+      <SubType>Designer</SubType>
+    </None>
```

Comment:
This change looks like an automatic change that's not part of the PR's goal. Can you please revert it?

Reply:
And done!